harfbuzz-sys/build.rs

```diff
@@ -1,17 +1,15 @@
+#[cfg(feature = "use-pkg-config-harfbuzz")]
+fn find_harfbuzz_via_pkgconfig() {
+    pkg_config::probe_library("harfbuzz").is_ok();
+}
+
 #[cfg(feature = "build-native-harfbuzz")]
-fn main() {
+fn build_harfbuzz() {
```

Comment:
What is the meaning of `native` here? I understand `native` to mean the version of HarfBuzz that comes with the system...but it's very confusing so maybe we should rename it.

Reply:
No idea anymore. It is confusing and I'd rather remove all of it really.